### ISSUE TITLE
cannot be restarted

### ISSUE THREAD

**Issue by AndreasEK** [geerlingguy/ansible-role-mysql#183]
See https://github.com/scrum-for-developers/scrum-for-developers-infra/issues/4

apparently that command seems to be run as root

**Comment by donat-b** [comment]
Root is required
https://github.com/geerlingguy/ansible-role-mysql#requirements

**Comment by geerlingguy** [comment]
@AndreasEK - Typically this role is run with `become: yes` (either on the full playbook level, or in the role line, e.g.:

```yaml
---
- hosts: database
  ...
  roles:
    - name: geerlingguy.mysql
      become: yes
```

**Comment by AndreasEK** [comment]
I think we did that:
https://github.com/scrum-for-developers/scrum-for-developers-infra/blob/master/ansible/provision-ec2-playbook.yml

**Comment by jp1987** [comment]
This is still an issue in ansible 2.2.0.0, see this issue (and referenced issued) for more info: https://github.com/ansible/ansible/issues/17490

I don't think it will hurt to add an explicit `become: yes` to the handler

**Comment by geerlingguy** [comment]
@jp1987 - Just out of curiousity... does the rest of the role work without a `become`? It seems like the apt installation, the root-owned file modifications, etc. would all break as well... it's puzzling that adding `become: yes` only to the handler would fix any issues unless MySQL is already installed and configured outside this role's use.

**Comment by jp1987** [comment]
It does. This is the way I include my role in my playbook:

`    - role: geerlingguy.mysql
      become: yes`

But there's a bug in Ansible (the one I linked, also in the referenced issues) where handlers do not inherit the global become parameter.

**Comment by jp1987** [comment]
I just upgrade to 2.2.1.0 (was running 2.2.0.0) and it works now (without the fix from the PR)

**Comment by geerlingguy** [comment]
@jp1987 - Good to know! In that case, I'll close the PR, and I think this issue should be resolved by upgrading to Ansible 2.2.1.
